packages/netlify-cms-widget-markdown/src/MarkdownControl/plugins/List.js

```diff
@@ -268,9 +268,46 @@ function ListPlugin({ defaultType, unorderedListType, orderedListType }) {
 
         if (listOrListItem.type === 'list-item') {
           const listItem = listOrListItem;
-
+          console.log({ listItem });
+          const {
+            value: { document },
+          } = editor;
           // If focus is at start of list item, unwrap the entire list item.
           if (editor.atStartOf(listItem)) {
+            const parentList = document.getParent(listItem.key);
+            /* If the list item in question has a grandparent list, this means it is a child of a nested list.
+             * Htting Enter key on an empty nested list item like this should move that list item out of the nested list
+             * and into the gradparent list. The targeted list item becomes direct child of its grandparent list
+             * Example
+             * <ul> ----- CLOSEST ANCESTOR LIST
+             *  <li> ------ CLOESET ANCESTOR LIST ITEM
+             *    <p>foo</p>
+             *    <ul> ----- PARENT LIST
+             *      <li>
+             *        <p>bar</p>
+             *      </li>
+             *      <li>
+             *        <p></p> ----- WHERE THE ENTER KEY HAPPENS
+             *      </li>
+             *    </ul>
+             *  </li>
+             * </ul>
+             */
+            const closestAncestorList = document.getClosest(
+              parentList.key,
+              block => block.type === (LIST_TYPES[0] || LIST_TYPES[1]),
```

Comment:
```suggestion
              block => LIST_TYPES.includes(block.type),
```

I'm not sure if the original code was supposed to be `block.type === LIST_TYPES[0] || block.type === LIST_TYPES[1]`